images/stackstorm-1ppc/entrypoint-1ppc.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+ST2_CONF=/etc/st2/st2.conf
+crudini --set ${ST2_CONF} auth    api_url     ${ST2_API_URL}
+crudini --set ${ST2_CONF} mistral api_url     ${ST2_API_URL}
+crudini --set ${ST2_CONF} mistral v2_base_url ${ST2_MISTRAL_API_URL}
+
+crudini --set ${ST2_CONF} api allow_origin '*'
+
+
+case "$ST2_SERVICE" in
+  "nop" )
+    exec tail -f /dev/null
+    ;;
+  "st2api" )
+    DAEMON_ARGS="-k eventlet -b 0.0.0.0:9101 --workers 1 --threads 1 --graceful-timeout 10 --timeout 30"
+    exec /opt/stackstorm/st2/bin/gunicorn_pecan /opt/stackstorm/st2/lib/python2.7/site-packages/st2api/gunicorn_config.py $DAEMON_ARGS
```

Comment:
These also came from original scripts in `/etc/init` like https://github.com/StackStorm/st2-packages/blob/v2.2/packages/st2/debian/st2.st2api.upstart and tried to use the exact same line (I made some necessary changes, though)

I just want to keep that way to make it reproducible for future version up.